packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -373,7 +385,7 @@ async function _getLastQueriedBlockNumber(configIdentifier, chainId) {
         const [dataField] = await datastore.get(key);
         return dataField[chainId];
       } else if (hubConfig.saveQueriedBlock == "localStorage") {
-        return process.env[`lastQueriedBlockNumber-${chainId}-${configIdentifier}`];
+        return process.env[`lastQueriedBlockNumber-${chainId}-${configIdentifier}`] | 0;
```

Comment:
when running locally can have an issue when there is no config stored. should return 0 in this case.